packaging/prover/postinst

```diff
@@ -21,6 +21,10 @@ then
 else
     mkdir -p /etc/opt/miden-prover
     sudo chown -R miden-prover /etc/opt/miden-prover
+    cat << EOF | sudo tee /etc/opt/miden-prover/service.conf
+    PORT=50052
+    TYPE=tx # [tx, batch, block]
```

Comment:
In the [docs](https://github.com/0xMiden/miden-base/tree/next/bin/proving-service#worker-configuration), we use `transaction` rather than `tx` - are these interchangeable?

Reply:
Also no longer relevant b/c of the env vars